front/entityconfig.form.php

```diff
@@ -0,0 +1,43 @@
+<?php
+/**
+ * --------------------------------------------------------------------------
+ * LICENSE
+ *
+ * This file is part of credit.
+ *
+ * credit is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * credit is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ * --------------------------------------------------------------------------
+ * @author    François Legastelois
+ * @copyright Copyright (C) 2017-2018 by Teclib'.
+ * @license   GPLv3 https://www.gnu.org/licenses/gpl-3.0.html
+ * @link      https://github.com/pluginsGLPI/credit
+ * @link      https://pluginsglpi.github.io/credit/
+ * -------------------------------------------------------------------------
+ */
+
+/** @file
+* @brief
+*/
+
+use Glpi\Event;
+
+include ('../../../inc/includes.php');
+
+Session::haveRight("entity", UPDATE);
```

Comment:
Could be better to be sure that we won't forget to update it if we define a dedicated rightname.
```suggestion
Session::haveRight(PluginCreditEntityConfig::$rightname, UPDATE);
```